planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -331,7 +335,11 @@ void planning_scene_monitor::PlanningSceneMonitor::scenePublishingThread()
           if (new_scene_update_ == UPDATE_SCENE)
             is_full = true;
           else
+          {
+            occupancy_map_monitor::OccMapTree::ReadLock lock;
+            if (octomap_monitor_) lock = octomap_monitor_->getOcTreePtr()->reading();
```

Comment:
Not important, but I dislike this mixed line format and it is also never used else where in MoveIt's code